File: Semestres/10.Agosto-Diciembre 2021/FADAUSB/11-Sep/Ejemplo2.py
"""
Autor: Carlos A Delgado
Fecha: 11 de Sep 2021
"""
import math

def recurrencia(n):
    if n==1:
        return 10
    else:
        return 2*recurrencia(n/8)+n**2

def formula(n):
    p1 = (-32/31)*(n**2)*(n**(-math.log2(32)/math.log2(8)) - 1)
    p2 = 10*n**(math.log2(2)/math.log2(8))
    return p1+p2


n = [8**x for x in range(0,15)]

for ni in n:
    print("n ",ni," RR ",recurrencia(ni)," Formula ",formula(ni))

File: Semestres/1.Febrero-Junio 2017/IPOO/Ejemplos/15Marzo/ejemplo2.cpp
/*
 * Autor: Carlos A Delgado
 * Archivo: Ejemplo1.cpp
 * Descripción: Ejemplo retorno void
 * Fecha: 15-Marzo-2017
 */
#include <iostream>

using namespace std;

int main(){
	
	int variable = 3;
	cout << variable << endl;
	cout << &variable << endl;
	
	cout << sizeof(variable) << endl;
	cout << sizeof(&variable) << endl;	
	
	int arreglo[100];
	
	cout << sizeof(arreglo) << endl;
	cout << sizeof(&arreglo) << endl;
	
	
}
